### DISCUSSION TITLE
Can the character set be limited for brute force?

### DISCUSSION THREAD

**Discussion by scottriffe-web** [illsk1lls/ZipRipper#46] [Q&A] [answered]
I believe that I know all of the characters in my zip password, but I can't remember the character order nor can I remember which ones are caps.  I've tried a few combinations, but no luck.

**Comment by illsk1lls** [comment, marked as answer]
not yet but I will look at what is involved

in the meantime, JohnTheRipper's official GUI Johnny may be helpful for more advanced jobs: [https://openwall.info/wiki/john/johnny](https://openwall.info/wiki/john/johnny)

if it's something I can add here I will
